backend/typescript/types.ts

```diff
@@ -120,6 +120,21 @@ export type CreateCampDTO = Omit<
   campSessions: Omit<CampSessionDTO, "id" | "camp" | "campers" | "waitlist">[];
 };
 
+export type UpdateCampDTO = Omit<
+  CampDTO,
+  "id" | "formQuestions" | "campSessions"
+>;
+
+export type CreateCampSessionDTO = Omit<
+  CampSessionDTO,
+  "id" | "camp" | "campers" | "waitlist"
```

Comment:
We currently support passing in CampSessions and FormQuestions to construct them at once – but in practice only basic details like description/fee will be called, and CampSessions / FormQuestions will be added one by one in a later section of registration